### ISSUE TITLE
[TW-349] l10n translation utility improvements

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#384]
_Paul Beckingham on 2013-04-18T03:33:53Z says:_ 

 * Force en-US.h to be the first column, for reference purposes, as it is the authoritative copy.
* Use ISO 639-2 instead of ISO 639-1, or at least support both.  http://www.loc.gov/standards/iso639-2/php/code_list.php
* Support analysis of just one localization file againt the en-US.h reference.
* Change shebang to be just "#!/usr/bin/python2 -tt" with no extra spaces, and supporting python 2.x, not just "python".

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-04-18T03:33:53Z
Modified: 2014-02-09T00:53:17Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2013-06-09T20:17:40Z says:_ 

 Moving to 2.4.0 so we can focus on sync.

**Comment by taskwarrior** [comment]
_Renato Alves on 2014-01-28T14:07:50Z says:_ 

 All features implemented with exception of -tt on shebang since it's not supported by all versions of /usr/bin/env (my own included).

**Comment by taskwarrior** [comment]
_Renato Alves on 2014-01-28T17:00:19Z says:_ 

 New version now accepts:
* -single xx-XX

or
* -single xx-XX.h

as valid input.

Wiki [[L10n]] was also updated accordingly.

**Comment by taskwarrior** [comment]
_Renato Alves on 2014-01-28T17:00:19Z says:_ 

 Attachment 0001-Feature-1255.patch has been added with description: Revised patch (obsoletes previous patch)

**Comment by taskwarrior** [comment]
_Renato Alves on 2014-01-28T18:05:23Z says:_ 

 Closed on commit c854196af0c8a8f94a38b23ac6484c0ebfb697ff

**Comment by pbeckingham** [comment]
[TW-349_0001-Feature-1255.patch.txt](https://github.com/GothenburgBitFactory/taskwarrior/files/1755583/TW-349_0001-Feature-1255.patch.txt)
